webapp/app/viewModels/role/roleAdd.html

```diff
@@ -0,0 +1,15 @@
+<section id="roleAdd">
+    <h1>Roles</h1>
+    <h3>Add</h3>
+    <div>
+        <label>New Role </label>
+        <span>
+            <input type="text" data-bind="value: name" />
+        </span>
+    </div>
+    <button type="button" data-bind="click: roleAdd">Add</button>
+    <div>
```

Comment:
Since you'll be redirecting the user to the role list after calling the API perhaps you can remove the response section